browser/ui/webui/settings/brave_extensions_manifest_v2_handler.cc

```diff
@@ -81,28 +55,31 @@ BraveExtensionsManifestV2Handler::BraveExtensionsManifestV2Handler() {
 
   // uBlock Origin
   extensions_.push_back(
-      {kUBlockId,
+      {extensions_mv2::kUBlockId, "https://github.com/gorhill/uBlock/",
        l10n_util::GetStringUTF16(
            IDS_SETTINGS_MANAGE_EXTENSIONS_V2_UBLOCK_ORIGIN_NAME),
        l10n_util::GetStringUTF16(
            IDS_SETTINGS_MANAGE_EXTENSIONS_V2_UBLOCK_ORIGIN_DESC),
        false});
 
   // uMatrix
-  extensions_.push_back({kUMatrixId,
+  extensions_.push_back({extensions_mv2::kUMatrixId,
+                         "https://github.com/gorhill/uMatrix/",
                          l10n_util::GetStringUTF16(
                              IDS_SETTINGS_MANAGE_EXTENSIONS_V2_UMATRIX_NAME),
                          l10n_util::GetStringUTF16(
                              IDS_SETTINGS_MANAGE_EXTENSIONS_V2_UMATRIX_DESC),
                          false});
 
   // AdGuard
-  extensions_.push_back({kAdGuardId,
-                         l10n_util::GetStringUTF16(
-                             IDS_SETTINGS_MANAGE_EXTENSIONS_V2_ADGUARD_NAME),
-                         l10n_util::GetStringUTF16(
-                             IDS_SETTINGS_MANAGE_EXTENSIONS_V2_ADGUARD_DESC),
-                         false});
+  extensions_.push_back(
+      {extensions_mv2::kAdGuardId,
+       "https://github.com/AdguardTeam/AdguardBrowserExtension",
```

Comment:
why other links has a trailing slash, but this one does not?

Reply:
removed everywhere